accounts-db/src/cache_hash_data.rs

```diff
@@ -339,19 +300,20 @@ impl CacheHashData {
         let entries = entries.iter().sum::<usize>();
         let capacity = cell_size * (entries as u64) + std::mem::size_of::<Header>() as u64;
 
-        let mmap = CacheHashDataFile::new_map(&cache_path, capacity)?;
+        let file = OpenOptions::new()
+            .create_new(true)
+            .write(true)
+            .open(cache_path)?;
+        let mut fw = BufWriter::new(file);
```

Comment:
I guess we should probably not go this route now?

Reply:
yeah. closed.